prob_rocket_sgra.py

```diff
@@ -209,46 +226,46 @@ def calcGrads(sizes,x,u,pi,constants):
         sinAlfa = numpy.sin(u[k,0])
         cosAlfa = numpy.cos(u[k,0])
         
-        beta = u[k,1]
-
+        cosu2 = numpy.cos(u2[k])

```

Comment:
What? u2 is a "dimensionless thrust", why are you computing the cosine? 
Makes no sense...